### ISSUE TITLE
Initialize database with info from another database

### ISSUE THREAD

**Issue by georgeef** [moneymanagerex/moneymanagerex#6887]
Proposed new feature: when a new database in created, the user has the option to initialize it with info from another database.

Info is any data except stocks and (scheduled) transactions, which change often. Initialization of all available info is roughly equivalent to a copy of the whole database and removal of stocks and (scheduled) transactions, together with records depending on them (e.g., split transactions, tags, attachments, custom fields).

## Configuration dialog

- Select the path of the original database.

- Select the tables to be initialized from the original database.

- Copy all selected tables.

## Notes

- Records in the new database may have different IDs than in the original database. The implementation may need to generate ID translation maps for several database tables.

**Comment by jensb** [comment]
Just thinking ...: Is this something that users need so often that MMEX needs an explicit feature for it?
Users can get the same result now, by copying the database and clearing all transactions tables. (Right?)
This takes an extra 5 minutes for the user but how often is this actually needed?

**Comment by georgeef** [comment]
Yes, it can also be done by copying everything and then removing what is not needed. This feature adds some convenience:
- Select if currencies, categories, payees, tags, etc., will be copied or not (users may want to copy only some of them).
- Set the initial balance of accounts to the current balance from the origin (in case the new file is a follow-up from the previous).

If the new database is mostly the same as the previous, copy and small cleanup is faster and simpler. If the new database needs only little information from the original, then cleanup is not so fast.
